app/models/content_item.rb

```diff
@@ -46,7 +46,7 @@ def self.create_or_replace(base_path, attributes)
   field :_id, :as => :base_path, :type => String, :overwrite => true
   field :content_id, :type => String
   field :title, :type => String
-  field :description, :type => String
+  field :description, :type => Hash, :default => { "value" => nil }
```

Comment:
It will now result in `nil`, yes.

Did it return a blank string before? I assumed that because there was no default, it would return `nil`, but I'll check.